### PR TITLE
Adjust review generation for TIPP import

### PR DESCRIPTION
**Pull request by hornmo**
Draft for new TIPP import review types, pending UI updates ...

### CODE REVIEW

**Comment by philboeselager** [comment]
Looks good to me.

**Comment by hornmo** [comment]
Now includes fixes & changes for tickets (145, 195 & 196)

File: server/gokbg3/src/main/groovy/org/gokb/UpdatePkgTippsRun.groovy [hunk 1]
```diff
@@ -705,50 +836,40 @@ class UpdatePkgTippsRun {
...
+
+      //     if (tipp) {
+      //       tipps << tipp
+      //     }
+      //     else  {
+      //       log.warn("ES record TIPP ${it.uuid} does not exist!")
+      //     }
+      //   }
+      //   return tipps
+      // }
```
Comment (philboeselager) [inline server/gokbg3/src/main/groovy/org/gokb/UpdatePkgTippsRun.groovy:882]: Maybe we can add a comment on how long we are wanting to keep out-commented code in the future. Sometimes, these blocks get really old and every time you pass by you ask yourself: Should I delete that?

**Review by philboeselager — APPROVED** [review]

File: server/gokbg3/src/main/groovy/org/gokb/UpdatePkgTippsRun.groovy (hunk 1 above)
Comment (hornmo) [reply server/gokbg3/src/main/groovy/org/gokb/UpdatePkgTippsRun.groovy:882]: Great Idea, I added comments with version info.

**Pull request merged**